application/models/SurveyDynamic.php

```diff
@@ -964,13 +964,19 @@ public function getQuestionArray($oQuestion, $oResponses, $bHonorConditions, $su
                 $aQuestionAttributes['answervalueslabels'][$oScaleSubquestion->title] = isset($oScaleSubquestion->question) ? $oScaleSubquestion->question : null;
             }
         }
-        
+
         if ($oQuestion->type=='N' || ($oQuestion->parent_qid != 0 && $oQuestion->parents['type'] === "K")) {
             if (strpos($aQuestionAttributes['answervalue'], ".") !== false) { // Remove last 0 and last . ALWAYS (see \SurveyObj\getShortAnswer)
                 $aQuestionAttributes['answervalue'] = rtrim(rtrim($aQuestionAttributes['answervalue'], "0"), ".");
             }
         }
 
+        // If trying to retrieve main question ($getCommentOnly = false), retrieve comment in a new attribute
+        // Check if $getCommentOnly = false to avoid endless recursivity
+        if ($oQuestion->type=='O'&&!$getCommentOnly) {
```

Comment:
PSR-12 recommends (or implies?) space between `==`. It should be:

```php
if ($oQuestion->type == 'O' && !$getCommentOnly) {
```

Please adjust.

Reply:
Done